src/guide/views/dropdown/DropdownExample.jsx

```diff
@@ -26,7 +26,7 @@ export default class DropdownExample extends Component {
     }, {
       name: 'Berry',
     }, {
-      name: 'Chippewa',
+      name: 'Corn',
```

Comment:
Chippewa is the name of a Native American tribe, but everyone else is something edible... one of these things was not like the others. 😮